### PR TITLE
Feature/friend page

### PR DESCRIPTION
**Pull request by timicienio**

### CODE REVIEW

File: src/partials/basic/Button.js [hunk 1]
```diff
@@ -14,6 +15,9 @@ const getClassNames = (variant, color) => {
     case 'secondary-danger': {
       return 'btn bg-white border-gray-200 hover:border-gray-300 active:border-gray-400 text-red-500';
     }
+    case 'secondary-tertiary': {
```
Comment (timicienio) [inline src/partials/basic/Button.js:18]:
```suggestion
    case 'tertiary-default': {
```

File: src/partials/basic/UserItem.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                <svg className="w-4 h-4 fill-current text-red-500 flex-shrink-0" viewBox="0 0 16 16">
+                  <path d="M5 7h2v6H5V7zm4 0h2v6H9V7zm3-6v2h4v2h-1v10c0 .6-.4 1-1 1H2c-.6 0-1-.4-1-1V5H0V3h4V1c0-.6.4-1 1-1h6c.6 0 1 .4 1 1zM6 2v1h4V2H6zm7 3H3v9h10V5z" />
+                </svg>
+              </button>
+              <button className="btn-sm border-gray-200 hover:border-gray-300">
+                <svg className="w-4 h-4 fill-current text-indigo-500 flex-shrink-0" viewBox="0 0 16 16">
+                  <path d="M14.3 2.3L5 11.6 1.7 8.3c-.4-.4-1-.4-1.4 0-.4.4-.4 1 0 1.4l4 4c.2.2.4.3.7.3.3 0 .5-.1.7-.3l10-10c.4-.4.4-1 0-1.4-.4-.4-1-.4-1.4 0z" />
+                </svg>
+              </button>
+            </div>
```
Comment (timicienio) [inline src/partials/basic/UserItem.js:33]: Use `<Button />`

File: src/partials/basic/UserItem.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        >
+          <div className="flex items-center">
+            <div className="relative mr-2">
+              <Avatar name={userName} />
+            </div>
+            <div className="truncate">
+              <span className="text-sm font-medium text-gray-800">{userName}</span>
+            </div>
+          </div>
+        </button>
```
Comment (timicienio) [inline src/partials/basic/UserItem.js:49]: Use `<Button />`

File: src/partials/friend/FriendBody.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+          className="md:hidden absolute top-4 left-4 sm:left-6 text-white opacity-80 hover:opacity-100"
+          onClick={() => setFriendSidebarOpen(!friendSidebarOpen)}
+          aria-controls="profile-sidebar"
+          aria-expanded={friendSidebarOpen}
+        >
+          <span className="sr-only">Close sidebar</span>
+          <svg className="w-6 h-6 fill-current text-gray-500" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+            <path d="M10.7 18.7l1.4-1.4L7.8 13H20v-2H7.8l4.3-4.3-1.4-1.4L4 12z" />
+          </svg>
+        </button>
```
Comment (timicienio) [inline src/partials/friend/FriendBody.js:28]: Use `<Button />`

File: src/partials/friend/FriendBody.js [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+            <h1 className="text-2xl text-gray-800 font-bold">icheft</h1>
+          </div>
+          {/* Bio */}
+          <div className="text-sm mb-3">Nothing more than a fitness fanatic.</div>
+          {/* Meta */}
+          <div className="flex flex-wrap justify-center space-x-4">
+            <div className="flex items-center">
+              <svg className="w-4 h-4 fill-current flex-shrink-0 text-gray-400" viewBox="0 0 16 16">
+                <path d="M11 0c1.3 0 2.6.5 3.5 1.5 1 .9 1.5 2.2 1.5 3.5 0 1.3-.5 2.6-1.4 3.5l-1.2 1.2c-.2.2-.5.3-.7.3-.2 0-.5-.1-.7-.3-.4-.4-.4-1 0-1.4l1.1-1.2c.6-.5.9-1.3.9-2.1s-.3-1.6-.9-2.2C12 1.7 10 1.7 8.9 2.8L7.7 4c-.4.4-1 .4-1.4 0-.4-.4-.4-1 0-1.4l1.2-1.1C8.4.5 9.7 0 11 0ZM8.3 12c.4-.4 1-.5 1.4-.1.4.4.4 1 0 1.4l-1.2 1.2C7.6 15.5 6.3 16 5 16c-1.3 0-2.6-.5-3.5-1.5C.5 13.6 0 12.3 0 11c0-1.3.5-2.6 1.5-3.5l1.1-1.2c.4-.4 1-.4 1.4 0 .4.4.4 1 0 1.4L2.9 8.9c-.6.5-.9 1.3-.9 2.1s.3 1.6.9 2.2c1.1 1.1 3.1 1.1 4.2 0L8.3 12Zm1.1-6.8c.4-.4 1-.4 1.4 0 .4.4.4 1 0 1.4l-4.2 4.2c-.2.2-.5.3-.7.3-.2 0-.5-.1-.7-.3-.4-.4-.4-1 0-1.4l4.2-4.2Z" />
+              </svg>
```
Comment (timicienio) [inline src/partials/friend/FriendBody.js:56]: Use `<Avatar />`

File: src/css/additional-styles/avatar.css [hunk 6]
```diff
@@ -15,6 +15,21 @@
...
 
+.initials-avatar.profile {
+	width: 82px;
+	height: 82px;
+}
+
+.initials-avatar.xl {
+	width: 64px;
+	height: 64px;
+}
```
Comment (timicienio) [inline src/css/additional-styles/avatar.css:26]: Looks like font sizes needs adjustment with these sizes

image

File: src/partials/basic/UserItem.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+          type="button"
+        >
+          <div className="flex justify-between group">
+            <div className="flex items-center">
+              <div className="relative mr-2">
+                <Avatar name={userName} />
+              </div>
+              <div className="truncate">
+                <span className="text-sm font-medium text-gray-800">{userName}</span>
+              </div>
```
Comment (timicienio) [inline src/partials/basic/UserItem.js:20]: These can be replaced with `<AvatarAndName />` component in the future.

File: src/partials/basic/UserItem.js (hunk 2 above)
Comment (icheft) [reply src/partials/basic/UserItem.js:33]: + [x] icons added

File: src/partials/basic/UserItem.js (hunk 3 above)
Comment (icheft) [reply src/partials/basic/UserItem.js:49]: not applicable

File: src/partials/friend/FriendBody.js (hunk 4 above)
Comment (icheft) [reply src/partials/friend/FriendBody.js:28]: not applicable

File: src/partials/friend/FriendBody.js (hunk 5 above)
Comment (icheft) [reply src/partials/friend/FriendBody.js:56]: not applicable

**Pull request merged**